views/pages/fullEpisodes.ejs

```diff
@@ -0,0 +1,17 @@
+
+<ul>
+    <li><a href="/">Home</a></li>
+    <li><a href="/fullEpisodes">Full Episodes</a></li>
+    <li><a href="/Contact">Contact Us</a></li>
+</ul>
```

Comment:
This is almost identical to the list of links on index.ejs.  perhaps they could share this list by way of including a partial.  (That might be something to research as an easy next step).  That would make the lists consistent going forward.